### ISSUE TITLE
SVG Native and Accessibility APIs

### ISSUE THREAD

**Issue by plehegar** [w3c/svgwg#874]
[SVG Native](https://svgwg.org/specs/svg-native/#commonattributes) removes support for ARIA attributes.

This removes any possible interaction with platform accessibility APIs. It is necessary to evaluate this specification against the [FAST checklist](https://w3c.github.io/apa/fast/checklist.html) before moving this document forward.

cc @michael-n-cooper @brewerj @caribouW3

**Comment by michael-n-cooper** [comment]
I'm not sure removing support for ARIA attributes stops all interaction with platform accessibility APIs. There are still [default mappings](https://www.w3.org/TR/svg-aam-1.0/#mapping_role_table) which I assume would still apply. If that is false, then that issue would be a big concern. I do not know what trade-offs are involved in removing ARIA attributes, so that aspect would need review as well. Copying @AmeliaBR as editor of SVG-AAM and @jnurthen as chair of ARIA WG. APA will also pick this up in its horizontal review issues check.

**Comment by michael-n-cooper** [comment]
Will bring up with ARIA WG

**Comment by michael-n-cooper** [comment]
Clarified with ARIA chair, the scope and reductions of SVG native mean it does not need the removed ARIA markup.

**Comment by michael-n-cooper** [comment]
APA continued to discuss, is worried that maybe it removes more than it needs to for the use cases. @matatk to review.

**Comment by matatk** [comment]
APA comment on the SVG Native Editor's Draft at: https://svgwg.org/specs/svg-native/

We are concerned that removing the ability to specify an accessible name and description for the image (i.e. `<title>` and `<desc>`) would present accessibility barriers, and would limit the capabilities of SVG Native compared to other image formats.

We're concerned about the lack of ability to specify an accessible name and description because content authors would have to take extra steps to provide that information when working in the native context versus on the web. By allowing the use of `<title>` and `<desc>` in SVG Native, authors could use the same file in both contexts. Many operating environments provide a platform accessibility API, and accessibility properties onto which the `<title>` and `<desc>` values could be mapped.

Other image formats provide for similar metadata: [the PNG spec provides for various metadata, including an image description](https://www.w3.org/TR/PNG/#4Concepts.AncillInfo); GIF and JPEG files provide for metadata and text comment storage.

We request that, when the underlying platform supports the exposure of accessible names or descriptions, that an SVG Native renderer (RFC 2119) MUST expose their contents to the accessibility API.

*[Record of the group's approval of this comment](https://lists.w3.org/Archives/Public/public-apa-admin/2022May/0008.html)*
